resources/js/package.js

```diff
@@ -61,10 +61,12 @@ function init() {
             loadingCount: 0,
             loading: false,
             loadAutocomplete: false,
+            openAgreement: null,
```

Comment:
Can we move this somewhere else? As it isn't a global shop thing.

Reply:
What would you suggest? Could turn it into a vue component but that seems unnecessarily bulky.

Reply:
Cant you dynamically handle this in the template?

Reply:
Nope, not without breaking reactivity. The entire reason I need to define it somewhere is so that vue can pick up on the variable and make it properly reactive.

Reply:
Maybe store it in a hidden input? Or use a radio to know which one is open?

Reply:
What I've done for now is change the slideover so that it accepts `v-bind:id`, which means we can use one slideover per agreement again.